screens/Scan/ScanLayout.tsx

```diff
@@ -48,6 +50,25 @@ export const ScanLayout: React.FC = () => {
             options={{
               title: t('sharingVc'),
               headerBackVisible: false,
+              headerTitle: props => (
+                <View
+                  style={{
```

Comment:
Can't we move this style to Theme class?

Reply:
moved to Theme files